doc/classes/LineEdit.xml

```diff
@@ -1,10 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="LineEdit" inherits="Control" category="Core" version="3.2">
 	<brief_description>
-		Control that provides single line string editing.
+		Control that provides single-line string editing.
 	</brief_description>
 	<description>
-		LineEdit provides a single line string editor, used for text fields.
+		LineEdit provides a single-line string editor, used for text fields. It features many built-in shortcuts which will always be available:
+
```

Comment:
We should not use empty lines in the XML, as each line is parsed as a separate paragraph.